src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -171,6 +171,19 @@ namespace webnn_native { namespace ie {
             return transposeNode;
         }
 
+        // Transpose HWNC <=> NCHW
```

Comment:
@fujunwei I don’t know if writing 'HWNC' here will cause some confusion. Do you have any ideas?

Reply:
I tried to define an enum to extend MLInputOperandLayout. But got the follow info: 'integral constant expression must have integral or unscoped enumeration type'. So I defined an enum for transpose directly.